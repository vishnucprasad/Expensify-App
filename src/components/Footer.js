import React from 'react';

const Footer = () => (
    <div className="footer">
        <div className="content-container">
            <p className="footer__text">&copy; 2021 Expensify. All rights reserved | Developed by <a href="https://vishnucprasad.github.io" target="_blank">Vishnu C Prasad</a></p>
        </div>
    </div>
);

export default Footer;