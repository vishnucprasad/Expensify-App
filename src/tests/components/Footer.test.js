import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

test('should correctly render Footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});