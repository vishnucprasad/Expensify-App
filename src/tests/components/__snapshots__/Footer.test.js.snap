// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Footer 1`] = `
<div
  className="footer"
>
  <div
    className="content-container"
  >
    <p
      className="footer__text"
    >
      Â© 2021 Expensify. All rights reserved | Developed by 
      <a
        href="https://vishnucprasad.github.io"
        target="_blank"
      >
        Vishnu C Prasad
      </a>
    </p>
  </div>
</div>
`;
